content/docs/rfcs/24/README.md

```diff
@@ -0,0 +1,134 @@
+---
+slug: 24
+title: 24/WAKU2-DNS-DISCOVERY
+name: Waku v2 Peer Discovery via DNS
```

Comment:
Is this Waku v2 specific? As opposed to being used BY Waku v2 in https://rfc.vac.dev/spec/10/#discovery-domain

Reply:
You're right. It's more general than Waku v2. I've renamed it to `Libp2p Peer Discovery via DNS`, which is more accurate. 57d7e62